README.md

```diff
@@ -64,18 +64,18 @@ cfork({
   // logger what you want
 });
 
-// if you do not listen to this event
-// cfork will listen it and output the error message to stderr
-process.on('uncaughtException', function (err) {
-  // do what you want
-});
-
 // emit when reach refork times limit
 .on('reachReforkLimit', function () {
   // do what you want
 });
 ```
 
+// if you do not listen to this event
```

Comment:
you can review your change https://github.com/jpuncle/cfork/blob/c3f28c55e7bf2d77ebb8e7ce88c558e762bcba1e/README.md